package.json

```diff
@@ -5,7 +5,7 @@
   "displayName": "HashiCorp HCL",
   "description": "HashiCorp HCL syntax",
   "appInsightsKey": "885372d2-6f3c-499f-9d25-b8b219983a52",
-  "version": "0.1.1",
+  "version": "0.1.0",
```

Comment:
👀

Reply:
Opps. Artifact of having to increment to test installation